VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/joh-waiting-room/joh-waiting-room.component.html

```diff
@@ -122,20 +122,19 @@ <h2 class="govuk-heading-m">We're having technical issues</h2>
       </div>
     </div>
     <div>
-      <a
-        href="javascript:void(0)"
+      <button
         class="govuk-button govuk-button--bigblue"
-        role="button"
-        draggable="false"
+        type="submit"
```

Comment:
I'd recommend removing the `type="submit"` - it doesn't have any effect when used outside of a form. Actually, even within a form 'submit' is the default action, so if there's only one button it isn't required even inside a form.

Reply:
Unless you've added is specifically for something like automated UI tests?